docs/tutorials/01_basic_example.ipynb

```diff
@@ -231,12 +252,74 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
    "id": "ac4be2b0-f1be-46db-a42d-9f204727db7b",
-   "metadata": {},
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-23T05:33:47.703182Z",
+     "start_time": "2024-06-23T05:33:47.698228Z"
+    }
+   },
+   "source": [
+    "my_algorithm_qref = {\"version\": \"v1\", \"program\": my_algorithm}"
+   ],
    "outputs": [],
+   "execution_count": 21
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ef92aec51534839",
+   "metadata": {},
    "source": [
-    "my_algorithm_qref = {'version': 'v1', 'program': my_algorithm}"
+    "So, is there an intuitive way to understand what my algorithm looks like and how the resources flow in the routine? You can use the visualization tool from [`QREF`](https://github.com/PsiQ/qref) to plot the hierarchical Directed Acyclic Graph (DAG) of the algorithm you wrote."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7d1b18a9c90420c",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info admonition note\"> <p class=\"admonition-title\"><b>NOTE:</b></p>\n",
+    "To use <a href=\"https://github.com/PsiQ/qref\">qref</a>'s redendering tool in Jupyter Notebook, ensure it's installed by executing the following command: <br>\n",
+    "<code> !pip install qref </code> <br>\n",
+    "Additionally, make sure the Graphviz software in your OS and ensure its executables are on your PATH system variable. (For installation instructions, please refer <a href=\"https://graphviz.org/download/\">here</a> for installation.)\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4f5318f14af6421",
+   "metadata": {},
+   "source": [
+    "Then, run: <br>\n",
+    "\n",
+    "<pre><code>\n",
+    "from qref import SchemaV1\n",
```

Comment:
I'd modify this slightly.
Namely do:
```
from qref.experimental.rendering import to_graphviz

# Validate the schema and convert to Graphviz object
gv_object = to_graphviz(my_algorithm_qref)

# Render the Graphviz object to a PNG file
gv_object.render("my_algorithm.png", format="png")

# Render the Graphviz object in the notebook
gb_object
```

Reply:
And, as we discussed, this should be executable snippet, rather than text as it is now.

Reply:
Resolved.